doc/install/easy-install.md

```diff
@@ -53,3 +53,8 @@ To pull the GPU version:
 ```bash
 docker pull ghcr.io/deepmodeling/deepmd-kit:2.0.0_cuda10.1_gpu
 ```
+
+To pull the ROCm version:
+```bash
+docker pull deepmodeling/dpmdkit-rocm:dp2.0.4-rocm4.5.2-tf2.6-lmp29Sep2021
```

Comment:
Why is it dp2.0.4?

Reply:
![image](https://user-images.githubusercontent.com/45301969/150485534-28c22805-28f8-49b1-9f28-352a8620d216.png)

I‘m confused, too...

Reply:
What's the source code you built from? setuptools-scm will bump a patch version if you do some change to a stable version, but you can use an environment variable to assign the version.

Reply:
It explains the matter~I have made some adjustments because the gelu gradient definition conflicts with TF 2.6.

Reply:
Also cmakefiles need to be modified on ROCm 3.5 or higher.